proj/bulgarian.md

```diff
@@ -0,0 +1,278 @@
+# Outdoor Relay Control
```

Comment:
I decided to add the full readme :)

Since Outdoor Realy Control (**ORC**) is the name of the project, I decided not to translate it.

Reply:
Thats good, because a name can't be translated...